hibernate-core/src/test/java/org/hibernate/jpa/test/criteria/basic/PredicateTest.java

```diff
@@ -246,6 +246,7 @@ public void testByteArray() {
 
 	@Test
 	@TestForIssue( jiraKey = "HHH-5803" )
+	@RequiresDialectFeature(DialectChecks.SupportsMixedTypeArithmetic.class)
```

Comment:
You'd better leave empty chars after '(' and before ')' as elsewhere.

Reply:
I will change this, but this does not seem very consistent throughout the Hibernate project (especially for annotations).